Client/src/Pages/AdvancedSettings/index.jsx

```diff
@@ -180,21 +239,37 @@ const AdvancedSettings = ({ isAdmin }) => {
 						</Typography>
 					</Box>
 					<Stack gap={theme.spacing(20)}>
-						<Field
-							type="text"
-							id="jwtTTL"
+						<Stack direction="row" gap={theme.spacing(10)} ><Field
+							type="number"
+							id="jwtTTLNum"
 							label="JWT time to live"
-							name="jwtTTL"
-							value={localSettings.jwtTTL}
+							name="jwtTTLNum"
+							value={localSettings.jwtTTLNum.toString()}
 							onChange={handleChange}
+							onBlur={handleBlur}
+							error={errors.jwtTTLNum}
 						/>
+						<Select
+							id="jwtTTLUnits"
+							label="JWT TTL Units"
+							name="jwtTTLUnits"
+							placeholder="Select time"
+							isHidden={true}
```

Comment:
Why are we declaring isHidden true?

Reply:
The placeholder "Select time" will be a hidden value